src/main/msp/msp.c

```diff
@@ -1544,9 +1544,9 @@ case MSP_NAME:
         sbufWriteU16(dst, gpsRescueConfig()->returnAltitudeM);
         sbufWriteU16(dst, gpsRescueConfig()->descentDistanceM);
         sbufWriteU16(dst, gpsRescueConfig()->groundSpeedCmS);
-        sbufWriteU16(dst, gpsRescueConfig()->throttleMin);
-        sbufWriteU16(dst, gpsRescueConfig()->throttleMax);
-        sbufWriteU16(dst, positionControlConfig()->hover_throttle);
+        sbufWriteU16(dst, autopilotConfig()->throttle_min);
```

Comment:
We need to consider creating separate MSP commands for apConfig but would become more clear when adding more ap functions later on.

Reply:
Best left till later, I agree.  Unless essential for some reason.